README.md

```diff
@@ -80,12 +78,12 @@ These instructions were tested on macOS 10.15 to 12, on Intel macs.
 
 	```shell
 	# Install some pre-flight dependencies.
-	brew install cairo calibre git openjdk pipx python
+	brew install cairo calibre git openjdk pipx python@3.11
```

Comment:
Will this stomp on some other globally-installed Python version?

Reply:
No, Homebrew’s happy to install multiple different concurrent python versions.